docs/source/faq/index.rst

```diff
@@ -49,6 +49,33 @@ App will launch/update in the output of your notebook cells!
 Check out the :ref:`notebook environment guide <notebooks>` for more
 information about running FiftyOne in notebooks.
 
+.. _faq-remote-notebook-support:
+
+Can I use FiftyOne in a remote notebook?
+----------------------------------------
+
+Yes! A common notebook workflow is to `launch jupyter on a remote system and
```

Comment:
comment on link style: can you hyperlink a smaller portion of the text so it doesn't dominate visually when scrolling through the FAQ? I believe in equal opportunity for all links :)